test/lib/css_coverage_test.rb

```diff
@@ -45,8 +36,4 @@ def setup
   def test_classify_does_not_generate_primer_css_classes_that_do_not_exist
     assert_empty(@classes_from_utilities - @css_data - @allowed_missing_classes_for_now)
   end
-
-  def test_docs_do_not_generate_primer_css_classes_that_do_not_exist
-    assert_empty(@classes_from_docs_build - @css_data - @allowed_missing_classes_for_now)
-  end
```

Comment:
This test was looking for classes being used in the docs that weren't in the primer/css bundle. It's beginning to fail more as we transition to #1319 So I'm removing that part of the test